### DISCUSSION TITLE
Quickdraw achievment

### DISCUSSION THREAD

**Discussion by krau5** [community/community#27460] [Other Feature Feedback, Questions, & Ideas] [answered]
If I have successfully merged pull-request in public repository, which was merged during first 5 minutes ( 2 minutes actually ), how much time is needed for quickdraw achievment to appear in my profile?

**Comment by kurnakovv** [comment]
@krau5 Quickdraw achievement not about PR, this is about issues, just create new issue in repository, after that close it.

If I helped you, please mark as answer :)

**Reply by krau5** [reply]
https://github.com/Schweinepriester/github-profile-achievements
<img width="890" alt="image" src="https://user-images.githubusercontent.com/52201020/181508309-23526531-3ddd-4d41-bf13-f3855a6d0025.png">

**Comment by fernandounger** [comment]
Hey @krau5 

- Quickdraw: appear when you create a new PR and they are lost within 5 minutes of opening.

![image](https://user-images.githubusercontent.com/87918106/181506726-c13bd311-f39d-4de9-a903-962bd28308a9.png)

**Reply by krau5** [reply]
Wdym by `lost`?

**Reply by jfullstackdev** [reply]
lost in that context means, you should close an issue or pull request within 5 minutes, lost because you will not be seeing it by default on those tabs

**Comment by krau5** [comment, marked as answer]
Okay, I've finally got it. Achievment won't unlock if you have contributed to someone's repository and its maintainer has approved and merged your PR within 5 minutes. It only works when you are merging it by yourself
